src/components/Layout/Header/SideNav.tsx

```diff
@@ -1,11 +1,19 @@
 import { chakra, useColorModeValue } from '@chakra-ui/react'
+import { useEffect, useState } from 'react'
+import { useWallet } from 'hooks/useWallet/useWallet'
 
 import { SideNavContent } from './SideNavContent'
 
 export const SideNav = () => {
   const bg = useColorModeValue('white', 'gray.850')
   const borderColor = useColorModeValue('gray.100', 'gray.750')
-
+  const {
+    state: { walletInfo },
+  } = useWallet()
+  const [top, setTop] = useState('4.5rem')
+  useEffect(() => {
+    setTop(walletInfo?.deviceId === 'DemoWallet' ? '7rem' : '4.5rem')
+  }, [walletInfo?.deviceId])
   return (
```

Comment:
Do you really need a useEffect and whatever?

I guess if you direclty use `walletInfo?.deviceId` as a ternary, it will rerender everytime it change, without the need of a useEffect hook?

Reply:
Much nicer, thanks mate!